localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_dark_iron_1;Golem Manufactory;;Golem-Manufaktur;;;;;;;;;;;x
+ca_culture_dark_iron_2;Golem Manufactory;;Golem-Manufaktur;;;;;;;;;;;x
+ca_culture_dark_iron_3;Golem Manufactory;;Golem-Manufaktur;;;;;;;;;;;x
+ca_culture_dark_iron_4;Golem Manufactory;;Golem-Manufaktur;;;;;;;;;;;x
+ca_culture_dark_iron_desc;Powerful machines of war, the war golems manufactured by the dark iron dwarves are truly a beguiling wonder to behold. Powered in part by fuel and magic, these golems can tear through enemy defenses, bulldoze entire hovels, and instill terror with the pounding of their metallic feet.;;Die von den dunkeleisernen Zwergen hergestellten m�chtigen Kriegsmaschinen, die Kriegsgolems, sind ein wahrhaft faszinierendes Wunderwerk. Diese Golems, die teilweise durch Treibstoff und Magie angetrieben werden, k�nnen feindliche Verteidigungsanlagen durchbrechen, ganze Geb�ude einrei�en und durch das Stampfen ihrer Metallf��e Schrecken einfl��en.;;;;;;;;;;;x
+ca_culture_night_elf_1;Sentinel Stronghold;;Festung der W�chter;;;;;;;;;;;x
+ca_culture_night_elf_2;Sentinel Stronghold;;Festung der W�chter;;;;;;;;;;;x
+ca_culture_night_elf_3;Sentinel Stronghold;;Festung der W�chter;;;;;;;;;;;x
+ca_culture_night_elf_4;Sentinel Stronghold;;Festung der W�chter;;;;;;;;;;;x
+ca_culture_night_elf_desc;Many night elves learn to use weapons before they can walk, or so it is said, but their natural talent in archery is almost unrivaled. The Sentinel Army's archers are excellent hunters, but during wartime they become critical support troops, firing upon their enemies to give their allies a chance to regroup and counterattack. They are renowned for their ambush ability and skills in guerrilla warfare gained by training in the forests of Kalimdor. They maximize their ability by using speed, terrain and dark-vision to their full advantage. "One shot, one kill.";;Viele Nachtelfen lernen den Umgang mit Waffen, bevor sie laufen k�nnen, so hei�t es, aber ihr nat�rliches Talent im Bogenschie�en ist unvergleichlich. Die Bogensch�tzen der W�chterarmee sind ausgezeichnete J�ger, aber in Kriegszeiten werden sie zu entscheidenden Unterst�tzungstruppen, die ihre Feinde unter Beschuss nehmen, um ihren Verb�ndeten die Chance zu geben, sich neu zu gruppieren und einen Gegenangriff zu starten. Sie sind bekannt f�r ihre Hinterhalte und ihre F�higkeiten in der Guerillakriegsf�hrung, die sie durch ihr Training in den W�ldern von Kalimdor erworben haben. Sie optimieren ihre F�higkeiten, indem sie Geschwindigkeit, Gel�nde und Nachtsicht zu ihrem vollen Vorteil nutzen. �Ein Schuss, ein Toter.�;;;;;;;;;;;x
```

Comment:
Changed `ability` to `tactics` and  `talents`
```suggestion
ca_culture_night_elf_desc;Many night elves learn to use weapons before they can walk, or so it is said, but their natural talent in archery is almost unrivaled. The Sentinel Army's archers are excellent hunters, but during wartime they become critical support troops, firing upon their enemies to give their allies a chance to regroup and counterattack. They are renowned for their ambush tactics and skills in guerrilla warfare gained by training in the forests of Kalimdor. They maximize their talents by using speed, terrain and dark-vision to their full advantage. "One shot, one kill.";;Viele Nachtelfen lernen den Umgang mit Waffen, bevor sie laufen k�nnen, so hei�t es, aber ihr nat�rliches Talent im Bogenschie�en ist unvergleichlich. Die Bogensch�tzen der W�chterarmee sind ausgezeichnete J�ger, aber in Kriegszeiten werden sie zu entscheidenden Unterst�tzungstruppen, die ihre Feinde unter Beschuss nehmen, um ihren Verb�ndeten die Chance zu geben, sich neu zu gruppieren und einen Gegenangriff zu starten. Sie sind bekannt f�r ihre Hinterhalte und ihre F�higkeiten in der Guerillakriegsf�hrung, die sie durch ihr Training in den W�ldern von Kalimdor erworben haben. Sie optimieren ihre F�higkeiten, indem sie Geschwindigkeit, Gel�nde und Nachtsicht zu ihrem vollen Vorteil nutzen. �Ein Schuss, ein Toter.�;;;;;;;;;;;x
```

Reply:
👍